src/docs/blueprint_documentation.md

```diff
@@ -143,6 +143,10 @@ isBound(AboutDialog) ? rebind(AboutDialog).to(TheiaBlueprintAboutDialog).inSingl
 
 To use another custom dialog widget, remove this code, extend Theia’s AboutDialog class, and (re)bind it as above.
 
+### Customizing the Preferences
+
+The default preferences directory in Eclipse Theia is `.theia-blueprint` and located as described in the [Preferences documentation](https://theia-ide.org/docs/preferences/). You can customize this location by modifying `theia-extensions/theia-blueprint-product/src/node/theia-blueprint-variables-server.ts`
```

Comment:
Bit of a nitpick, but I think this should be "in Eclipse Theia Blueprint", since in Theia it would be just `.theia`.

Reply:
Thanks, updated and rebased.